02_what_why_intro.Rmd

```diff
@@ -1,20 +1,107 @@
+
+Exploratory visualations and EDA allow for researchers to explore their data and validate that it is the correct data to answer the research question. A lot of time will be spent with exploratory visualization and EDA not because you're making any one plot really pretty and very understandable. In fact, you should spend very little time with each plot. The large amount of time instead is the result of making a lot of visualizations and thoroughly exploring the dataset. The time you take at the beginning of an analysis to explore the dataset and truly understand and wrangle what is there, saves a lot of time in the end because you can request more data or make any necessary corrections before you perform the dataset and build up your expository visualizations.
+
+### Expository Visualizations
+
+Expository visualizations are polished visualizations prepared for the purpose of sharing. You may share an expository visualization within a preprint, a peer-reviewed publication, on a poster, or as part of a presentation.
+
+Because expository visualizations will be shared widely, you will not always be there to present or explain the visualization to others. Therefore, you want to design your data visualization to **stand alone**.
+
+With this goal in mind, an important part of creating an expository visualization also includes writing the narrative to go along with it. This narrative may involve an informative title, well-labeled axes, a thorough caption, and alternative text. Note that not all formats require each of these elements to appear alongside the visualization. You won't put a caption on your presentation slide, however you may want to include the caption within the speaker notes.
+
+<details><summary>Specific tips for a high quality narrative</summary>
+
+* Ensure the title states the plot's main conclusion
+* Avoid using field specific jargon or obscure acronyms within a title or an axis label unless defined (perhaps within the caption).
+
+</details>  
+
+
+Additionally, because an expository visualization will be shared widely, the audience will be broad and have varied interests and expertise. If you are giving a semi-frequent presentation within your department or lab group or presenting within a session at a specialized conference, you can expect the audience to be somewhat familiar with the topic. However, no one will know your research like you do. Many others may see the visualization outside of your department or institution -- perhaps even patients or their family members! Therefore, you want to prepare your expository visualizations in a way that they will be accessible to a larger audience.
```

Comment:
```suggestion
Additionally, because an expository visualization will be shared widely, the audience will be broad and have varied interests and expertise. If you are giving a semi-frequent presentation within your department or lab group or presenting within a session at a specialized conference, you can expect the audience to be somewhat familiar with the topic. However, no one will know your research like you do. Many others may see the visualization outside of your lab, department, or institution -- perhaps even patients and their family members, or other members of the public! Therefore, you want to prepare your expository visualizations in a way that they will be accessible to a larger audience.
```